episodes/imaging-software.md

```diff
@@ -0,0 +1,486 @@
+specific types of image data or processing steps. The availability of a specific 
+plugin will often be a deciding factor on whether to use Fiji/ImageJ or Napari 
+for your project.
+
+
+**Ease of installation and user interface**  
+As Fiji/ImageJ has been in development for longer, it tends to be simpler to 
+install than Napari (especially for those with no prior Python experience). In 
+addition, as it has more built-in image processing tools, it tends to be simpler 
+to use fully from its user interface. Napari meanwhile is often strongest when 
+you combine it with some Python scripting (although this isn't required for many 
+workflows!)
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+For this lesson, we will use Napari as our software of choice. It's worth 
+bearing in mind though that Fiji/ImageJ can be a useful alternative - and many 
+workflows will actually use both Fiji/ImageJ and Napari together! Again, it's 
+about choosing the right tool for your data and research question.
+
+
+## Opening Napari
+
+Let's get started by opening a new Napari window - you should have already 
+followed the [installation instructions](../learners/setup.md). Note this can 
+take a while the first time, so give it a few minutes!
+
+```bash
+conda activate napari-env
+napari
+```
+
+![](fig/blank-napari-ui.png){alt="A screenshot of the default Napari user 
+interface"}
+
+## Opening images
+
+Napari comes with some example images, let's open one now:
+
+```
+Go to the top menu-bar of Napari and select:
+File > Open Sample > napari builtins > Cells (3D+2Ch)
+```
+
+You should see a fluorescence microscopy image of some cells:
+
+![](fig/cells-napari.png){alt="A screenshot of a flourescence microscopy image 
+of some cells in Napari"}
+
+## Napari's User interface
+
+Napari's user interface is split into a few main sections, as you can see in the 
+diagram below:
+
+![](fig/ui-sections-napari.png){alt="A screenshot of Napari with the main user 
+interface sections labelled"}
+
+Let's take a brief look at each of these sections - for full information see the 
+[Napari documentation](https://napari.org/stable/tutorials/fundamentals/viewer.html).
+
+## Main menu
+
+We already used the main menu in the last section to open a sample image. The 
+main menu contains various commands for opening images, changing preferences and 
+installing plugins (we'll see more of these options in later episodes).
+
+## Canvas
+
+The canvas is the main part of the Napari user interface. This is where we 
+display and interact with our images. 
+
+Try moving around the cells image with the following commands:
+```
+Pan - Click and drag
+Zoom - Scroll in/out
+```
+
+## Dimension sliders
+
+Dimension sliders appear at the bottom of the canvas depending on the type of 
+image displayed. For example, if we drag the slider at the bottom of the cells 
+image, we move up and down in this 3D image stack.
```

Comment:
```suggestion
Dimension sliders appear at the bottom of the canvas depending on the type of 
image displayed. Here we have a 3D image, which consists of a stack of 2D images. If we drag the slider at the bottom of the cells image, we move up and down in this stack of 2D images.
```
Since we have not yet talked about 2D and 3D, I thought it would be important to give a little more context here.